Documentation/filesystem.md

```diff
@@ -120,28 +123,61 @@ You now have a docker registry which is HA with persistent storage.
 #### Kernel Version Requirement
 If the Rook cluster has more than one filesystem and the application pod is scheduled to a node with kernel version older than 4.7, inconsistent results may arise since kernels older than 4.7 do not support specifying filesystem namespaces.
 
-## Test the storage
+## Consume the Shared File System: Toolbox and the Ceph tools
 
 Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared file system in the toolbox pod.
+This is just a simple way to validate the Ceph file system and is not recommended for production Kubernetes pods.
 
-Start and connect to the [Rook toolbox](toolbox.md).
+Start and connect to the [Rook toolbox](toolbox.md) to mount the same filesystem that the kube-registry is using.
 
 ```bash
-# Mount the same filesystem that the kube-registry is using
+# Create the directory
 mkdir /tmp/registry
-rookctl filesystem mount --name myfs --path /tmp/registry
```

Comment:
not a fan of how much more complicated these manual steps got without `rookctl`! RIP `rookctl` :(

Reply:
Yes, simplicity was lost for the manual steps for sure.